react/src/pages/VFolderNodeListPage.tsx

```diff
@@ -0,0 +1,527 @@
+import BAITabs from '../components/BAITabs';
+import Flex from '../components/Flex';
+import FolderCreateModal from '../components/FolderCreateModal';
+import ImportFromHuggingFaceModal from '../components/ImportFromHuggingFaceModal';
+import InviteFolderPermissionSettingModal from '../components/InviteFolderPermissionSettingModal';
+import QuotaPerStorageVolumePanelCard from '../components/QuotaPerStorageVolumePanelCard';
+import StorageStatusPanelCard from '../components/StorageStatusPanelCard';
+import VFolderNodes, { VFolderNodeInList } from '../components/VFolderNodes';
+import {
+  filterNonNullItems,
+  handleRowSelectionChange,
+  transformSorterToOrderString,
+} from '../helper';
+import { useUpdatableState } from '../hooks';
+import { useBAIPaginationOptionState } from '../hooks/reactPaginationQueryOptions';
+import { useCurrentProjectValue } from '../hooks/useCurrentProject';
+import { useDeferredQueryParams } from '../hooks/useDeferredQueryParams';
+import {
+  VFolderNodeListPageQuery,
+  VFolderNodeListPageQuery$data,
+  VFolderNodeListPageQuery$variables,
+} from './__generated__/VFolderNodeListPageQuery.graphql';
+import { useToggle } from 'ahooks';
+import {
+  Badge,
+  Button,
+  Col,
+  Grid,
+  Row,
+  theme,
+  Tooltip,
+  Typography,
+} from 'antd';
+import graphql from 'babel-plugin-relay/macro';
+import _ from 'lodash';
+import React, { Suspense, useDeferredValue, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLazyLoadQuery } from 'react-relay';
+import { StringParam, withDefault } from 'use-query-params';
+
+type VFolderNodesType = NonNullableNodeOnEdges<
+  VFolderNodeListPageQuery$data['vfolder_nodes']
+>;
+
+const VFOLDER_STATUSES = [
+  'READY',
+  'PERFORMING',
+  'CLONING',
+  'MOUNTED',
+  'ERROR',
+  'DELETE_PENDING',
+  'DELETE_ONGOING',
+  'DELETE_COMPLETE',
+  'DELETE_ERROR',
+];
+
+interface VFolderNodeListPageProps {}
+
+const VFolderNodeListPage: React.FC<VFolderNodeListPageProps> = ({
+  ...props
+}) => {
+  const { t } = useTranslation();
+  const { token } = theme.useToken();
+  const { lg } = Grid.useBreakpoint();
+  const currentProject = useCurrentProjectValue();
+
+  const [selectedFolderList, setSelectedFolderList] = useState<
+    Array<VFolderNodesType>
+  >([]);
+  const [inviteFolderId, setInviteFolderId] = useState<string | null>(null);
+  const [isOpenCreateModal, { toggle: openCreateModal }] = useToggle(false);
+  const [
+    isVisibleImportFromHuggingFaceModal,
+    { toggle: toggleImportFromHuggingFaceModal },
+  ] = useToggle(false);
+  const {
+    baiPaginationOption,
+    tablePaginationOption,
+    setTablePaginationOption,
+  } = useBAIPaginationOptionState({
+    current: 1,
+    pageSize: 10,
+  });
+
+  const [queryParams, setQuery] = useDeferredQueryParams({
+    order: StringParam,
+    filter: StringParam,
+    statusCategory: withDefault(StringParam, 'created'),
+    mode: withDefault(StringParam, 'all'),
+  });
```

Comment:
Let's preserve/restore the status based on the `statusCategory` tab. It's the same way as in [ComputeSessionListPage](https://github.com/lablup/backend.ai-webui/blob/88f1c2bed2cc0164891ded0068e7e33159419af5/react/src/pages/ComputeSessionListPage.tsx#L66-L68).